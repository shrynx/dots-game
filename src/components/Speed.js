// @flow
import styled from 'styled-components';
import { PURPLE } from '../styles/colors';

const Speed = styled.span`
  font-size: 12px;
  text-align: center;
  font-weight: 600;
  color: ${PURPLE};
`;

export default Speed;
