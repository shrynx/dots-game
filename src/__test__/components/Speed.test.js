import React from 'react';
import { render } from 'react-testing-library';
import Speed from '../../components/Speed';

it('Speed component should render correctly', () => {
  const { container } = render(<Speed>5</Speed>);
  expect(container).toMatchSnapshot();
});
