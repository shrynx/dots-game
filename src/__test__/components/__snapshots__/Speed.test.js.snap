// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Speed component should render correctly 1`] = `
<div>
  <span
    class="sc-bdVaJa jNIuGw"
  >
    5
  </span>
</div>
`;
